app/src/main/resources/ui/helper-foreign-keys.js

```diff
@@ -47,7 +50,8 @@ async function createForeignKeyLinksFromPlan(newForeignKey, foreignKey, linkType
     for (const fkLink of Array.from(foreignKey[`${linkType}Links`])) {
         let newForeignKeyLink = await createForeignKeyInput(numForeignKeysLinks, `foreign-key-${linkType}-link`);
         foreignKeyLinkSources.insertBefore(newForeignKeyLink, foreignKeyLinkSources.lastChild);
-        $(newForeignKeyLink).find(`select.foreign-key-${linkType}-link`).selectpicker("val", fkLink.taskName);
+        $(newForeignKeyLink).find(`select.foreign-key-${linkType}-link`).selectpicker("val", fkLink.taskName)[0].dispatchEvent(new Event("change"));
```

Comment:
Should check if the result is non-empty before calling the first element via `[0]`.